### PR TITLE
Update wip-learning.md

### PR DESCRIPTION
**Pull request by loadedleke**
The path the App protection status has changed. The image on the page also needs to be changed. See below.
image

### CODE REVIEW

**Comment by cabailey** [comment]
I have no knowledge of this product - looking for correct writer ownership

**Comment by cabailey** [comment]
Reassigning to dougeby as agreed - please update the writer/manager fields for this article (I don't work in this repo)

File: windows/security/information-protection/windows-information-protection/wip-learning.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ In the **Website learning report**, you can view a summary of the devices that h
 
 1. Sign in to the [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
 
-1. Click **Client apps** > **App protection status** > **Reports**.
+1. Click **Apps** > **Monitor** > **App protection status** > **Reports**.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/wip-learning.md:36]:
```suggestion
1. Select **Apps** > **Monitor** > **App protection status** > **Reports**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @loadedleke, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by aczechowski — APPROVED** [review]
Thanks @loadedleke for making this change. I also updated the article metadata - I'm the best assignee for right now.

**Review by CryptoGodfatherVA2 — COMMENTED** [review]
confirmed

**Pull request merged**